synapse/lib/ast.py

```diff
@@ -2772,25 +2959,58 @@ async def getCondEval(self, runt):
         name = await self.kids[0].compute(runt, None)
 
         prop = runt.model.props.get(name)
-        if prop is None:
-            mesg = f'No property named {name}.'
-            raise self.kids[0].addExcInfo(s_exc.NoSuchProp(mesg=mesg, name=name))
+        if prop is not None:
+            if prop.isform:
```

Comment:
You could probably combine these two ifs and dedent the body.

```suggestion
        if prop is not None and prop.isform:
```

Reply:
There are actually two different conditions that are returned here if prop is not None, one if it is a form and a separate one if it is not a form